docs/reference/monitoring/collecting-monitoring-data.asciidoc

```diff
@@ -0,0 +1,213 @@
+
+... Create a user on the monitoring cluster that has the 
+{stack-ov}/built-in-roles.html[`remote_monitoring_agent` built-in role]. 
+Alternatively, use the 
+{stack-ov}/built-in-users.html[`remote_monitoring_user` built-in user].
+
+... Add the user ID and password settings to the HTTP exporter settings in the 
+`elasticsearch.yml` file on each node. +
++
+--
+For example:
+
+[source,yaml]
+--------------------------------------------------
+xpack.monitoring.exporters:
+  id1:
+    type: http
+    host: ["http://es-mon-1:9200", "http://es-mon2:9200"] 
+    auth.username: remote_monitoring_user 
+    auth.password: YOUR_PASSWORD
+--------------------------------------------------
+--
+
+.. If you configured the monitoring cluster to use 
+<<configuring-tls,encrypted communications>>, you must use the HTTPS protocol in 
+the `host` setting. You must also specify the trusted CA certificates that will 
+be used to verify the identity of the nodes in the monitoring cluster. 
+
+*** To add a CA certificate to an {es} node's trusted certificates, you can 
+specify the location of the PEM encoded certificate with the 
+`certificate_authorities` setting. For example:
++
+--
+[source,yaml]
+--------------------------------------------------
+xpack.monitoring.exporters:
+  id1:
+    type: http
+    host: ["https://es-mon1:9200", "https://es-mon2:9200"] 
+    auth:
+      username: remote_monitoring_user
+      password: YOUR_PASSWORD
+    ssl:
+      certificate_authorities: [ "/path/to/ca.crt" ]
+--------------------------------------------------
+--
+
+*** Alternatively, you can configure trusted certificates using a truststore
+(a Java Keystore file that contains the certificates). For example:
++
+--
+[source,yaml]
+--------------------------------------------------
+xpack.monitoring.exporters:
+  id1:
+    type: http
+    host: ["https://es-mon1:9200", "https://es-mon2:9200"]
+    auth:
+      username: remote_monitoring_user
+      password: YOUR_PASSWORD
+    ssl:
+      truststore.path: /path/to/file
+      truststore.password: password
+--------------------------------------------------
+--
+
+. Configure your cluster to route monitoring data from sources such as {kib}, 
+Beats, and {ls} to the monitoring cluster. The 
+`xpack.monitoring.collection.enabled` setting must be `true` on each node in the 
+cluster. For information about configuring each product to collect and send 
+monitoring data, see {stack-ov}/xpack-monitoring.html[Monitoring the {stack}].
+
+. If you updated settings in the `elasticsearch.yml` files on your production 
+cluster, restart {es}. See <<stopping-elasticsearch>> and <<starting-elasticsearch>>. 
++
+--
+TIP: You may want to temporarily {ref}/modules-cluster.html[disable shard
+allocation] before you restart your nodes to avoid unnecessary shard
+reallocation during the install process.
+
+--
+
+. Optional: 
+<<config-monitoring-indices,Configure the indices that store the monitoring data>>. 
```

Comment:
Isn't this information mentioned earlier already?

> .. Optional: Specify which indices you want to monitor.

Reply:
By my understanding, those steps are different.  One is specifying which indices to monitor, the other is tweaking the .monitoring-* indices. If that's not correct or not clear, let me know!

Reply:
You're right, I didn't read properly.